angular/src/products.html

```diff
@@ -0,0 +1,246 @@
+        font-size: 1em;
+    }
+    
+    .navbar {
+        border-bottom: 3px solid #58A3D5;
+        right: 0;
+        margin-bottom: 0em;
+        padding-top: 1em;
+        width: 100%;
+    }
+    
+    .navAnchors {
+        float: right;
+        font-size: 2em;
+        padding-bottom: 1em;
+        padding-right: 1em;
+        text-decoration: none !important;
+    }
+    
+    .navAnchor:hover {
+        color: #CDCDCD;
+        background-color: #CDCDCD;
+    }
+    
+    .navAnchor:before::hover {
+        color: #CDCDCD;
+        background-color: #CDCDCD;
+    }
+    
+    h3 {
+        color: #47a3da;
+    }
+    
+    .widget {
+        margin-top: 2.2em;
+        font-size: 2em;
+        color: #58A3D6;
+    }
+    
+    span {
+        color: #C0C0C0;
+        font-size: 2em;
+        display: block;
+    }
+    
+    .list-price {
+        margin-top: 2em;
+    }
+    
+    .description {
+        margin-top: 2.6em;
+        color: #59A4D7;
+    }
+    
+    strong {
+        font-stretch: condensed;
+    }
+    
+    .listimg {
+        padding: 0;
+        border: 0.5em solid lightgray;
+        margin-bottom: 2em;
+        display: block;
+    }
+    
+    listingimg:hover {
+        border: 0.5em solid grey;
+    }
+    
+    button {
+        margin-top: 3em;
+        color: white;
+        background-color: #47a3da;
+        border-radius: 2px;
+        height: 3em;
+        padding: 1em;
+    }
+    
+    button:hover {}
+    
+    button > a {
+        color: white;
+    }
+    
+    .listli {
+        padding-bottom: 1em;
+    }
+</style>
+
+
+<body ng-app="BetterThanEtsy">
+
+    <div class="container">
+
+        <nav class="navbar">
+            <!--
+           <ul class="nav nav-pills col-xs-offset-8 col-sm-offset-10 col-md-offset-10" role="tablist">
+               <li role="presentation" class="active">
+-->
+            <a class="navAnchors glyphicon glyphicon-th-list" href="#list" role="tab" data-toggle="tab"></a>
+
+
+            <!--
+               </li>
+               <li role="presentation">
+-->
+            <a class="navAnchors glyphicon glyphicon-th" href="#grid" role="tab" data-toggle="tab"></a> <span>&nbsp;&nbsp;</span>
+            <!--
+            </li>
+</ul>
+-->
+        </nav>
+
+        <section class="tab-content" ng-controller="StoreController as store">
+            <div class="tab-pane active" id="grid">
+                <!--this div is for the grid layout-->
+                <div ng-repeat="item in store.items" class="col-xs-12 col-sm-6 col-md-4 text-center">
+                    <img class="text-center" ng-src="{{item.MainImage[0].url_75x75}}">
+                    <h1>{{item.title}}</h1>
+                    <span>{{item.price}}</span>
+                    <p>{{item.description}}</p>
+                    <button>Add to cart</button>
+                </div>
+
+            </div>
+            <!--this div is for the grid layout-->
+
+            <div class="tab-pane" id="list">
```

Comment:
We refactored it today and we are now using Angular. Tomorrow we will be working on refactoring Html and Css into Sass and hopefully we will be able to use Ng-Class to show the two different views.